File: examples/node/get.ts
import { SailupClient } from '@kalourmade/sailup-sms';

const messageId = process.argv[2];
if (!messageId) {
  console.error('Usage: tsx get.ts <message_id>');
  process.exit(1);
}

const client = new SailupClient(process.env.SAILUP_API_KEY!);
const message = await client.sms.get(messageId);

console.log(`${message.id} — ${message.body} (${message.deliveryStatus})`);

File: examples/python/get.py
import os
import sys

from kalourmade_sailup import SailupClient

message_id = sys.argv[1] if len(sys.argv) > 1 else sys.exit("Usage: python get.py <message_id>")
client = SailupClient(os.environ["SAILUP_API_KEY"])

message = client.sms.get(message_id)

print(f"{message.id} — {message.body} ({message.delivery_status.value})")
